project_national/fact_sheets_category_6.yml

```diff
@@ -0,0 +1,752 @@
+        type: CO2e
+        elec_folder: data/cambium/LRMER_MidCase_15
+
+    utility_bills:
+      - scenario_name: Bills
+
+    simulation_output_report:
+      timeseries_frequency: timestep #monthly
+      include_timeseries_total_consumptions: true
+      include_timeseries_fuel_consumptions: true
+      include_timeseries_end_use_consumptions: true
+      include_timeseries_emissions: true
+      include_timeseries_total_loads: false
+
+    reporting_measures:
+      - measure_dir_name: QOIReport
+
+    server_directory_cleanup:
+      retain_in_idf: false
+      retain_schedules_csv: false
+
+
+### -- Upgrades ---
+# Category 6:  Heat Pumps with Existing Backup
+# To do: costing interpolation from Less for 9.2, 9.5, and 11 HSPF
+upgrades:
+  - upgrade_name: EnergyStar ducted ASHP w/ existing backup  #also provides SEER 15 cooling measure profile
+    # Measure 1
+    options:
+      #ASHP
+      - &ducted_ASHP_EnergyStar_exist_backup_no_shared_ducts
+        option: HVAC Heating Efficiency|ASHP, SEER 16, 9.2 HSPF, Separate Backup # same as base option
+        apply_logic: &dualfuel_ASHP_no_ducted_furnace_apply_logic
+          - HVAC Has Ducts|Yes
+          - or:
+            - Heating Fuel|Natural Gas
+            - Heating Fuel|Propane
+            - Heating Fuel|Fuel Oil
+          - not:
+              or:
+                - HVAC Heating Type And Fuel|Natural Gas Fuel Furnace
+                - HVAC Heating Type And Fuel|Propane Fuel Furnace
+                - HVAC Heating Type And Fuel|Fuel Oil Fuel Furnace
+                - HVAC Heating Efficiency|Shared Heating
+        costs: &costs_ashp_ducted_SEER_16_9-2_HSPF
+        # Source: Custom regression by Brennan Less. Ducted heat pump project costs were regressed on nameplate tons and HSPF,
+        # from data described in in Less et al. https://doi.org/10.20357/B7FP4D. 2019$ inflated to 2022$ using 1.12 factor.
+        # Ducted ASHP regression n=317. Implicitly includes electrical upgrades and electric backup.
+          - value: 3907.01 # for HSPF 9
+            multiplier: Fixed (1)
+          - value: 155.17
+            multiplier: Size, Heating System Primary (kBtu/h)
+        lifetime: &lifetime_ashp_ducted_SEER_16_9-2_HSPF 15
+
+  # Boilers as secondary systems
+  # Set secondary boiler fuel type
+      - &ng_boiler_secondary_heating_type_fuel
+        option: HVAC Secondary Heating Type And Fuel|Natural Gas Boiler
+        apply_logic:
+          - HVAC Heating Type And Fuel|Natural Gas Fuel Boiler
+      - &propane_boiler_secondary_heating_type_fuel
+        option: HVAC Secondary Heating Type And Fuel|Propane Boiler
+        apply_logic:
+          - HVAC Heating Type And Fuel|Propane Fuel Boiler
+      - &fuel_oil_boiler_secondary_heating_type_fuel
+        option: HVAC Secondary Heating Type And Fuel|Fuel Oil Boiler
+        apply_logic:
+          - HVAC Heating Type And Fuel|Fuel Oil Fuel Boiler
+  # Set secondary fuel boiler efficiency levels
```

Comment:
Looks like maybe you made a backup option for every boiler HVAC Heating Efficiency in options_lookup, but you only should need the 3 in the .tsv (ie 76%, 80%, 90%)